tests/test_handler_stats.py

```diff
@@ -26,74 +26,97 @@
 
 class TestHandlerStats(unittest.TestCase):
     def test_metrics_print(self):
-        log_stream = StringIO()
-        log_handler = logging.StreamHandler(log_stream)
-        log_handler.setLevel(logging.INFO)
-        key_to_handler = "test_logging"
-        key_to_print = "testing_metric"
+        def event_filter(_, event):
+            if event in [1, 2]:
+                return True
+            return False
+
+        for epoch_log in [True, event_filter]:
```

Comment:
@wyli can we use something like pytest parametrize here instead of for-loop ?

Reply:
thanks, I can think of this approach: https://github.com/vfdev-5/MONAI/pull/1